sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -310,6 +311,19 @@ def encode(self, value):
     return value.SerializePartialToString(deterministic=True)
 
 
+class ProtoPlusCoderImpl(SimpleCoderImpl):
```

Comment:
Do you think this could affect the performance characteristics of existing Python GCP connectors ?

Reply:
It shouldn't- either they're currently not resolving a coder (and broken) or theyre already registering a custom coder (and this is ignored) or they're not using proto-plus types in their interfaces.